README.md

```diff
@@ -1,19 +1,26 @@
 # Symfony CMF Resource Bundle
 
-[![Build Status](https://travis-ci.org/symfony-cmf/resource-bundle.svg?branch=master)](https://travis-ci.org/symfony-cmf/resource-bundle)
-[![StyleCI](https://styleci.io/repos/26688804/shield)](https://styleci.io/repos/26688804)
-[![Latest Stable Version](https://poser.pugx.org/symfony-cmf/resource-bundle/version.png)](https://packagist.org/packages/symfony-cmf/resource-bundle)
-[![Total Downloads](https://poser.pugx.org/symfony-cmf/resource-bundle/d/total.png)](https://packagist.org/packages/symfony-cmf/resource-bundle)
+[![Latest Stable Version](https://poser.pugx.org/symfony-cmf/resource-bundle/v/stable)](https://packagist.org/packages/symfony-cmf/resource-bundle)
+[![Latest Unstable Version](https://poser.pugx.org/symfony-cmf/resource-bundle/v/unstable)](https://packagist.org/packages/symfony-cmf/resource-bundle)
+[![License](https://poser.pugx.org/symfony-cmf/resource-bundle/license)](https://packagist.org/packages/symfony-cmf/resource-bundle)
 
-This bundle is part of the [Symfony Content Management Framework (CMF)](http://cmf.symfony.com/)
-and licensed under the [MIT License](LICENSE).
+[![Total Downloads](https://poser.pugx.org/symfony-cmf/resource-bundle/downloads)](https://packagist.org/packages/symfony-cmf/resource-bundle)
+[![Monthly Downloads](https://poser.pugx.org/symfony-cmf/resource-bundle/d/monthly)](https://packagist.org/packages/symfony-cmf/resource-bundle)
+[![Daily Downloads](https://poser.pugx.org/symfony-cmf/resource-bundle/d/daily)](https://packagist.org/packages/symfony-cmf/resource-bundle)
+
+Branch | Travis | Coveralls |
+------ | ------ | --------- |
+master | [![Build Status][travis_unstable_badge]][travis_unstable_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] |
+
+This package is part of the [Symfony Content Management Framework (CMF)](http://cmf.symfony.com/) and licensed
+under the [MIT License](LICENSE).
 
 This bundle provides *object* resource location services based on Puli.
 
 Examples:
 
 - **Static document path mapping**: Map the path `/routes` to `/cms/routes`
-- **Dynamic document resolution**: Map the path `/routes` to `/cms/<current site>/routes`.
+- **Dynamic document resolution**: Map the path `/routes` to `/cms/&lt;current site&gt;/routes`.
```

Comment:
i think its wrong to html encode here. when i look at the github rendered markdown file, it shows the code, not `<`.